source/less/blocks/main-nav.less

```
@@ -1,3 +1,34 @@
+.main-nav {
+  z-index: 100;
+  position: relative;
+
+  &.main-nav--closed {
+    .main-nav__wrapper {
+      display: none;
+    }
+  }
```

```suggestion
.main-nav {
  z-index: 100;
  position: relative;
}

.main-nav--closed {
  .main-nav__wrapper {
    display: none;
  }
}
```
В твоем варианте селектор `.main-nav.main-nav--closed` выглядит избыточным. Достаточно просто `.main-nav--closed`. К тому же это нарушение критерия Д11. Критерий дополнительный - можешь пока не тратить время на исправление.